### DISCUSSION TITLE
Stage uploads when device is offline

### DISCUSSION THREAD

**Discussion by LeoKlaus** [LeoKlaus/Paperparrot#63] [Feature Requests]
Allow users to queue uploads while the device is offline.
This only applies when the app is connected to a server, thus it's low priority for now.

**Comment by Br4thering** [comment]
In case the app is running standalone on MacOS I'd love to see an option to specify a folder which Paperparrot scans on launch for "waiting" documents (like the consume folder in nix). I use a scanner app on my phone which scans to a next cloud folder which is synced back to my MacBook. Would be handy for Paperparrot Standalone (without nix server connectivity) users if such a directory is automatically scanned on next application launch

**Reply by LeoKlaus** [reply]
Quick question regarding this:
Why don't you use the scanner within the Paperparrot app on your phone? This way you could skip the nextcloud folder.

Other than that, I think that's a neat idea. Probably a good chunk of work though, so I don't know when I'll find the time for this.

**Comment by sn0opy** [comment]
Similar to this. I had a local DNS hickup which caused the app not being able to resolve my paperless server's address thus it was not able to reach it at all. The app didn't tell me about it though. When I tried to scan + upload a document it simply closed the window without telling me if it was successful or not.

**Reply by LeoKlaus** [reply]
There's an issue for that already at #157. This is definitely on my todo list for the next release.
